Foreground.skin.php

```diff
@@ -199,11 +199,11 @@ public function execute() {
 		<div class="row">
 				<div id="p-cactions" class="large-12 columns">
 					<?php if ($wgUser->isLoggedIn() || $wgForegroundFeatures['showActionsForAnon']): ?>
-						<a href="#" data-dropdown="drop1" class="button dropdown small secondary radius"><i class="fa fa-cog"><span class="show-for-medium-up">&nbsp;<?php echo wfMessage( 'actions' )->text() ?></span></i></a>
+						<a href="#" class="button dropdown small secondary radius"><i class="fa fa-cog"><span class="show-for-medium-up">&nbsp;<?php echo wfMessage( 'actions' )->text() ?></span></i></a>
```

Comment:
Ok You also have to add hover_timeout:5000; or it will close as soon as you move the mouse. But when you try to close it by hovering out of the box there is a 5sec delay how can we automate this so it dose this only if we leave the box.